tests/driver/runner_test.py

```diff
@@ -823,7 +822,6 @@ def test_query_match_using_request_params(self, es):
 
         es.search.assert_called_once_with(
             index="_all",
-            doc_type=None,
```

Comment:
Would be it a lot of effort to add one unit test where `doc_type` is defined, say together with request_params?

Reply:
No problem, silly oversight on my part.